app/models/tag.rb

```diff
@@ -115,6 +115,10 @@ def self.find_nodes_by_type(tagnames, type = 'note', limit = 10)
         .order(order)
   end
 
+  def self.counter(tagname)
+    find_nodes_by_type(tagname, 'note', false).count + find_nodes_by_type(tagname, 'page', false).count + find_nodes_by_type("question:#{tagname}", 'note', false).count
```

Comment:
This looks really good. I do think we need to watch out a bit for performance issues on this, and will just ask for folks to reflect on performance once before we merge it -- see how the `find_nodes_by_type()` method is relatively heavy: 

https://github.com/publiclab/plots2/blob/80a649ab55b1ac574ac29ca3335ac98f80ccca60/app/models/tag.rb#L94-L116

And we're calling it ~4 times on this page then? Not a blocking concern but just one to watch out for. Thanks -- any thoughts on how to optimize or do folks think we can go ahead and just keep an eye on this page's load time using Skylight? (https://oss.skylight.io/app/applications/GZDPChmcfm1Q/recent/6h/endpoints)

Reply:
@jywarren should I create another method for this entirely not calling any existing method?

Reply:
I'm thinking it would be more efficient to query all nodes with that tagname with a single query instead of applying a type for each search. That way the query would ignore the type and we would only have to use one instead of 4. I'm not very familiar with how the tag organization works in the db. Do you know who could help with this @jywarren?

Reply:
yes indeed it would be fine to query once for all types. Good idea!